saleor/payment/gateways/stripe/plugin.py

```diff
@@ -456,7 +464,7 @@ def validate_plugin_configuration(cls, plugin_configuration: "PluginConfiguratio
                 raise ValidationError(
                     {
                         field: ValidationError(
-                            "The parameter is required.",
+                            "The parameter is required",
```

Comment:
Why removing the period? I think most of our messages have it 🤔  🚔

Reply:
Will bring it back as it is something that is returned by API.